packages/custom-functions-metadata/src/generate.ts

```diff
@@ -1150,3 +1190,16 @@ export function logError(error: string, position?: ts.LineAndCharacter | null):
   }
   return error;
 }
+
+function convertStringToBoolean(status: string): boolean {
+  return "true" == status.toLowerCase()
+}
+
+function convertIParseTreeResultIncludingFieldsToObject(parseTreeResultIncludingFields : IParseTreeResultIncludingFields) : Object {
+  let obj = Object.create(null);
+  obj["functions"] = parseTreeResultIncludingFields.functions;
+  for(let [k,v] of parseTreeResultIncludingFields.fields){
+    obj[k] = v;
+  }
+  return obj;
+}
```

Comment:
Should be a break line here. Please run `npm run lint:fix` on the package root.

Reply:
> Should be a break line here. Please run `npm run lint:fix` on the package root.

Ok.